backend/types/generated/contentTypes.d.ts

```diff
@@ -482,50 +482,6 @@ export interface PluginUploadFolder extends Schema.CollectionType {
   };
 }
 
-export interface PluginI18NLocale extends Schema.CollectionType {
```

Comment:
The folder with the name "GENERATED" is always generated. This explanation is only applicable in this case so if you meet folder name generated in other projects, it could be meaning different thing. In our case, all the files in the gerated folder will be auto generated.